webpack_loader/loader.py

```diff
@@ -85,6 +85,8 @@ def get_bundle(self, bundle_name):
         elif assets.get('status') == 'error':
             if 'file' not in assets:
                 assets['file'] = ''
+            if 'error' not in assets:
+                assets['error'] = 'error'
```

Comment:
I'm not sure about this but may be we can say something like `Unknown Error in ...` ?